SConstruct

```diff
@@ -215,6 +215,14 @@ opts.Add(
         False,
     )
 )
+opts.Add(
+    BoolVariable(
+        "profiler_track_memory",
+        "Profile memory allocations, if the profiler supports it.",
+        True,
```

Comment:
Since tracking memory leads to unexpected crashes, let's actually disable it by default for now.
We can re-enable it if we figure out what's going wrong.


```suggestion
        False,
```